vkcs/networking/data_source_subnet_test.go

```diff
@@ -146,56 +152,60 @@ func testAccCheckNetworkingSubnetDataSourceGoodNetwork(n1, n2 string) resource.T
 }
 
 const testAccNetworkingSubnetDataSourceSubnet = `
-resource "vkcs_networking_network" "network_1" {
-  name = "network_1"
+resource "vkcs_networking_network" "network_acc_test" {
+  name = "network_acc_test"
   admin_state_up = "true"
+  tags = [
+	"{{.TestName}}",
+	"{{.CurrentTime}}",
+  ]
 }
 
-resource "vkcs_networking_subnet" "subnet_1" {
-  name = "subnet_1"
+resource "vkcs_networking_subnet" "subnet_acc_test" {
+  name = "subnet_acc_test"
   description = "my subnet description"
   cidr = "192.168.199.0/24"
-  network_id = vkcs_networking_network.network_1.id
+  network_id = vkcs_networking_network.network_acc_test.id
   tags = [
-    "foo",
-    "bar",
+	"{{.TestName}}",
+	"{{.CurrentTime}}",
   ]
 }
 `
 
 const testAccNetworkingSubnetDataSourceBasic = `
 {{.TestAccNetworkingSubnetDataSourceSubnet}}
 
-data "vkcs_networking_subnet" "subnet_1" {
-  name = vkcs_networking_subnet.subnet_1.name
+data "vkcs_networking_subnet" "subnet_acc_test" {
+  name = vkcs_networking_subnet.subnet_acc_test.name
+  tags = vkcs_networking_subnet.subnet_acc_test.tags
 }
 `
 
 const testAccNetworkingSubnetDataSourceCidr = `
 {{.TestAccNetworkingSubnetDataSourceSubnet}}
 
-data "vkcs_networking_subnet" "subnet_1" {
+data "vkcs_networking_subnet" "subnet_acc_test" {
   cidr = "192.168.199.0/24"
-  tags = []
+  tags = vkcs_networking_subnet.subnet_acc_test.tags
 }
 `
 
 const testAccNetworkingSubnetDataSourceDhcpEnabled = `
 {{.TestAccNetworkingSubnetDataSourceSubnet}}
 
-data "vkcs_networking_subnet" "subnet_1" {
-  network_id = vkcs_networking_network.network_1.id
+data "vkcs_networking_subnet" "subnet_acc_test" {
+  network_id = vkcs_networking_network.network_acc_test.id
   dhcp_enabled = true
-  tags = [
-    "bar",
-  ]
+  tags = vkcs_networking_subnet.subnet_acc_test.tags
 }
 `
 
 const testAccNetworkingSubnetDataSourceGatewayIP = `
 {{.TestAccNetworkingSubnetDataSourceSubnet}}
 
-data "vkcs_networking_subnet" "subnet_1" {
-  gateway_ip = vkcs_networking_subnet.subnet_1.gateway_ip
+data "vkcs_networking_subnet" "subnet_acc_test" {
+  gateway_ip = vkcs_networking_subnet.subnet_acc_test.gateway_ip
+  tags = vkcs_networking_subnet.subnet_acc_test.tags
```

Comment:
I think that in the all cases above, maybe, except for searching by the name, it's more logical just to specify network_id in a data source.

Reply:
When  I specify network_id rest attributes may be ignored, so we discussed with @ftersin and decide to use unique tags